packages/website/docs/creating-multi-source-autocompletes.md

```diff
@@ -2,3 +2,375 @@
++                       <path fill="currentColor" d="M14,3V5H17.59L7.76,14.83L9.17,16.24L19,6.41V10H21V3M19,19H5V5H12V3H5C3.89,3 3,3.9 3,5V19A2,2 0 0,0 5,21H19A2,2 0 0,0 21,19V12H19V19Z" />
++                     </svg>
++                  </div>
++                  <div className="'aa-ItemTitle">{item.label}</div>
++                </a>
++              )
++        },
+        },
+      },
+    ];
+  },
+};
+```
+
+### Adding CSS for the predefined items
+
+To style the predefined links, create a `predefinedItemsPlugin.css` file in your `src` directory, and add this CSS to it:
+
+```css
+.aa-ItemLink {
+  color: inherit;
+  text-decoration: none;
+}
+
+```
+
+Don't forget to include this style sheet in your markup.
+
+### Adding the predefined items plugin to the autocomplete
+
+All that's left is to import the newly created `predefinedItemsPlugin` and add it to`plugins` in `index.js`. Once you've done that, the file should look like this:
+
+```diff
+import { autocomplete } from '@algolia/autocomplete-js';
++ import { predefinedItemsPlugin } from './predefinedItemsPlugin';
++
+// Instantiate the autocomplete instance
+autocomplete({
+  container: '#autocomplete',
+-  plugins: [],
++  plugins: [predefinedItemsPlugin],
+  openOnFocus: true,
+});
+```
+
+Now, as soon as a user clicks on the search bar, these predefined items appear. Once they begin typing, only predefined items that contain the query remain.
+
+<AutocompleteExample
+  plugins={[predefinedItemsPlugin]}
+  openOnFocus={true}
+/>
+
+## Adding recent searches and Query Suggestions
+
+You can add recent searches and Query Suggestions using out-of-the-box plugins. Refer to the guides on [adding recent searches](using-recent-searches-plugin) and [Query Suggestions](using-query-suggestions-plugin) for more detailed explanations.
+
+### Creating a recent searches plugin
+
+Use the out-of-the-box [`createLocalStorageRecentSearchesPlugin`](createlocalstoragerecentsearchesplugin) function to create a recent searches plugin:
+
+```js
+import { createLocalStorageRecentSearchesPlugin } from '@algolia/autocomplete-plugin-recent-searches';
+
+const recentSearchesPlugin = createLocalStorageRecentSearchesPlugin({
+  key: 'RECENT_SEARCH',
+  limit: 5,
+});
+
+```
+
+The `key` can be any string and is useful to differentiate search histories if you have multiple autocompletes on one page. The `limit` defines the maximum number of recent searches to display.
+
+### Creating a Query Suggestions plugin
+
+:::note
+
+**You must have a populated [Query Suggestions](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/js/#implementing-query-suggestions) index to use this plugin.** If you don't have a Query Suggestions index yet, follow the guide on [creating a Query Suggestions index](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/how-to/creating-a-query-suggestions-index/js/).
```

Comment:
How about providing the credentials from our demo index so that they can follow the guide with a dataset?

https://github.com/algolia/autocomplete.js/blob/8a4da0211d9f671efe1c26f1ec6179fb4a77ff1c/examples/js/app.tsx#L26-L38